src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -178,6 +179,10 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, route}: Workspac
         });
     }, [policy, isOffline, policyFeatureStates, prevPendingFields]);
 
+    useEffect(() => {
+        App.confirmReadyToOpenApp();
+    }, []);
+
```

Comment:
This is for when you open the App directly in a workspace's settings page:

<img width="1436" alt="image" src="https://github.com/user-attachments/assets/2dc5296c-58a2-4780-8bc1-9b4769608a58">